examples/alerts/alerts.md

```diff
@@ -117,8 +115,7 @@ rules:
     severity: critical
 - alert: ThanosRuleHighRuleEvaluationWarnings
   annotations:
-    description: Thanos Rule {{$labels.job}} {{$labels.pod}} has high number of evaluation
```

Comment:
What's the reason of `labels.pod` removal? It's because it's not always `pod`?

Reply:
As suggested I'll use `instance`.

Reply:
I removed them because I thought do we actually need to know the pod/instance in description